spec/ddtrace/contrib/mysql2/patcher_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'spec_helper'
+
+require 'ddtrace'
+require 'mysql2'
+
+RSpec.describe 'Mysql2::Client patcher' do
+  let(:tracer) { Datadog::Tracer.new(writer: FauxWriter.new) }
+
+  let(:client) do
+    Mysql2::Client.new(
+      host: host,
+      port: port,
+      database: database
+    )
+  end
+
+  let(:host) { ENV.fetch('TEST_MYSQL_HOST') { '127.0.0.1' } }
+  let(:port) { ENV.fetch('TEST_MYSQL_PORT') { '3306' } }
+  let(:database) { 'test' }
```

Comment:
I think this might need to be `let(:database) { ENV.fetch('TEST_MYSQL_DB', 'mysql') }` so as to avoid configuration issues in our test environment (which uses the default MySQL db name, which I don't think is `test`.)